test/auto/injectorSpec.js

```diff
@@ -237,6 +237,15 @@ describe('injector', function() {
     });
 
 
+    // Only Chrome and Firefox support this syntax.
+    if (/chrome|firefox/i.test(navigator.userAgent)) {
```

Comment:
I do not have access to MS Edge and it is currently not supported by saurcelabs, so this will have to make it.

On a side note, I find it somehow strange for MS Edge user agent to have the word "Chrome" in it